Bookish.Web/Controllers/HomeController.cs

```diff
@@ -18,10 +18,10 @@ public HomeController(ILogger<HomeController> logger, IBookishService bookishSer
             this.bookishService = bookishService;
         }
 
-        public IActionResult Catalogue(string filter = "")
+        public IActionResult Catalogue(int page = 1, string filter = "")
```

Comment:
What happens if the user manually changes the url to a page number greater than the number of books? Or changes it to 0 or a negative number? 🙂

Reply:
Page number too high: the webpage displays the same page as when the library has no books.
Page number too low: the table displays page 1 except the styling for the page numbers is incorrect.

While the website doesn't crash I suppose these should just return an error page?

Reply:
Either an error page or just correct the numbers so that page number too high -> display max page number, page number too low -> display first page. Up to you which one you want to go for 🙂